### ISSUE TITLE
It cannot convert this dictionary

### ISSUE THREAD

**Issue by viktorstrate** [Jason3S/hunspell-reader#2]
This is a continuation of this issue Jason3S/cspell-dicts#26
The danish dictionary for Libre Office [stavekontrolden.dk](http://www.stavekontrolden.dk/main/top/index.php) cannot be converted by hunspell-reader.
Direct links to dictionary: [da_DK.dic](http://www.stavekontrolden.dk/main/sspinputfiles/da_DK.dic), [da_DK.aff](http://www.stavekontrolden.dk/main/sspinputfiles/da_DK.aff)

When tried, the following is outputted to the terminal.
```bash
$ hunspell-reader words danish.dic

Write words
Sort: No
Unique: No
Ignore Case: No
Dic file: danish.dic
Aff file: danish.aff
Generating Words
```

I have tried exactly the same with an english .dic/.aff dictionary with no problems.
The [official hunspell cli](https://github.com/hunspell/hunspell) can use the danish dictionary without problems:

```bash
$ hunspell -d ./stavekontrolden ./file-to-check
```

I have tried to find error in the source code. And I think the problem lies in the `createFromFiles()` function in `HunspellReader.js`. I have added some debug console logs, and a .catch(), to catch the error:

```javascript
// dist/HunspellReader.js
static createFromFiles(affFile, dicFile) {
        console.log("createFromFiles")
        return affReader_1.parseAffFileToAff(affFile)
            .then(aff => {
                console.log("parseAffFileToAff done")
                return new HunspellSrcInfoWithGetDic(aff, () => HunspellReader.readDicEntries(aff, dicFile));
            })
            .then(src => new HunspellReader(src))
            .catch(err => {
                console.log("This is the error!", err)
            })
    }
```

When called with these changes, the terminal outputs the following

```bash
$ hunspell-reader words danish.dic

Write words
Sort: No
Unique: No
Ignore Case: No
Dic file: danish.dic
Aff file: danish.aff
Generating Words
createFromFiles
Error SyntaxError: Invalid regular expression: /+GENITIV$/: Nothing to repeat
    at new RegExp (<anonymous>)
    at tablePfxOrSfx (/usr/local/lib/node_modules/hunspell-reader/dist/affReader.js:66:23)
    at asSfx (/usr/local/lib/node_modules/hunspell-reader/dist/affReader.js:80:12)
    at ScanSubscriber.lines.pipe.operators_1.reduce [as accumulator] (/usr/local/lib/node_modules/hunspell-reader/dist/affReader.js:145:26)
    at ScanSubscriber._tryNext (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/operators/scan.js:118:27)
    at ScanSubscriber._next (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/operators/scan.js:111:25)
    at ScanSubscriber.Subscriber.next (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/Subscriber.js:103:18)
    at MapSubscriber._next (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/operators/map.js:92:26)
    at MapSubscriber.Subscriber.next (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/Subscriber.js:103:18)
    at FilterSubscriber._next (/usr/local/lib/node_modules/hunspell-reader/node_modules/rxjs/internal/operators/filter.js:96:30)
```

I don't understand the structure of the .aff files, but I can see that it contains `+GENETIV` in some places, here for an example, on line 446:

```
#Genitiv
SFX 70 Y 5
SFX 70 0 s/944 [^sxz]	+GENITIV
SFX 70 0 es/944 s	+GENITIV
SFX 70 0 ’s/944 s	+GENITIV
SFX 70 0 '/944 [sxz]	+GENITIV
SFX 70 0 es/944 s	+GENITIV
```

**Comment by Jason3S** [comment]
Looks like `+GENITIV` is getting pulled into the regex test.

The `+` must have a special meaning for Hunspell, so that it is not included in the replacement.

**Comment by Jason3S** [comment]
This should be fixed with 2.0.3

**Comment by viktorstrate** [comment]
Thank you, it looks like it's working 👍
